src/groupByKey.ts

```diff
@@ -2,6 +2,18 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function
```

Comment:
Function keyword and function name should be on the same line. It is better to change it to 'export function groupByKey'